app/scripts/constants.js

```diff
@@ -109,6 +109,17 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     namespace: "openshift"
   },
 
+  INACTIVITY_TIMEOUT: {
```

Comment:
@jwforres @spadgett here we need to align on the default time that will be set. Also I believe that the `is_active` variable should be set to **false**, right ?

Reply:
There probably only needs to be one value. I'd suggest letting the user set the timeout in minutes. Make any value that is not positive (undefined, null, 0, -1) could mean off? @jwforres opinion?

Default should be disabled, yes.

Ultimately this should go into the AssetConfig when we split the console into a pod and openshift-ansible updated to have a flag to set it. But that can happen in a later PR.

Reply:
Agreed, one variable. Any positive integer is on, anything else is off.

Reply:
So based on https://github.com/openshift/origin-web-console/pull/2528#discussion_r152555338 I thought that we want to be able to configure multiple varibles like those mentioned in https://github.com/openshift/origin-web-console/pull/2528#issuecomment-352446268, so we don't have to hardcode it into code.
In case we really want just a single variable that would set the inactivity period in minutes(none positive value would disable the inactivity logout), we need to align on following values that would need to be hardcoded:
**check_for_timeout_period** - Seconds or that shall console check for timeout - I guess it would be enough to check every minute ?
**modal_countdown_period** - Set to number of milliseconds that will be used in a for visual countdown in the modal - I've set it to 20 seconds in demo. Guess thats OK ?

Reply:
sorry i think there is some confusion here. I was just saying that you don't need `is_active` since setting `timeout_period` to something other than a positive integer would indicate whether we should be checking a timeout or not. And I agree with @spadgett that the timeout period should be configured in minutes.

I don't think you need `check_for_timeout_period`, checking once a minute for inactivity is sufficient. We will just document that it works that way.

For `modal_countdown_period` I would default to 30 seconds, that is what I commonly see.  I would also configure it in seconds, not milliseconds.  @spadgett were suggesting that we didn't need this one?

Reply:
> For modal_countdown_period I would default to 30 seconds, that is what I commonly see. I would also configure it in seconds, not milliseconds. @spadgett were suggesting that we didn't need this one?

Personally I don't see a ton of value in making it configurable and might cause some confusion in how the two values are different when an admin first configures it.

Reply:
Ok i'm fine with that. Let's just make it 30 seconds then and we'll need to document it.